spec/ttml3.xml

```diff
@@ -1410,6 +1427,24 @@ consist of one or more <loc href="#terms-inline-area">inline areas</loc>.</p>
 <p>Any of the element types defined by the <loc href="#element-vocab-type-metadata">Metadata Module</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-module">
+<label>[module]</label>
+<def>
+<p>A collection of element types, attributes, features, or the specification(s) of such entities. Modules may be defined
+<loc href="#terms-internal-module">internally</loc> (within the specification of an edition or version of a TTML defining document)
+or <loc href="#terms-external-module">externally</loc> (in one or more separate documents). Moreover, each module is designated as either
+<loc href="#terms-public-module">public</loc> or <loc href="#terms-private-module">private</loc>, according to whether it is listed
+in the <loc href="#terms-module-registry">module registry</loc> or not.</p>
+</def>
```

Comment:
Why is it necessary to differentiate between private and public modules? Specifically, how does this characteristics affect processing?

Reply:
Why is it necessary to define private use areas in Unicode? To provide a reserved space for unregistered characters. We need to account for unregistered modules, as an implementation might support one and an author might use one. The proposed definitions give us frame the terms that support a conceptual model of the space of modules, which includes those that aren't registered. Not everything in TTML affects conformance testable processing. Some of the content of TTML provide guidance about how to experiment outside the scope of standard defined functionality.